README.md

```diff
@@ -16,16 +16,16 @@ design](https://docs.google.com/document/d/1Oi9EO3Jwtp69obakl-9YpLkP764GZzsz95XJ
 
 ## Prerequisites
 
-You must have a recent verzion of [bazel](https://bazel.io) installed. Bazel is
+You must have a recent version of [bazel](https://bazel.io) installed. Bazel is
 the recommended way to build and test the cluster registry. Bazel is designed to
 maintain compatibility with standard Go tooling, but this is not tested on a
 regular basis, and some scripts/tooling in this project are built around Bazel.
```

Comment:
Can you add a note about needing hg installed here? Bazel does not take care of that for you, and I don't think it's expected to take care of this level of tool integration, so it's a prerequisite that must be installed separately.

Reply:
I added a note about `hg`. We can probably assume folks have `git` installed. :)